packages/flutter/lib/src/rendering/platform_view.dart

```diff
@@ -269,41 +269,24 @@ class RenderAndroidView extends PlatformViewRenderBox {
   }
 }
 
-/// A render object for an iOS UIKit UIView.
-///
-/// [RenderUiKitView] is responsible for sizing and displaying an iOS
-/// [UIView](https://developer.apple.com/documentation/uikit/uiview).
-///
-/// UIViews are added as sub views of the FlutterView and are composited by Quartz.
-///
-/// {@macro flutter.rendering.RenderAndroidView.layout}
-///
-/// {@macro flutter.rendering.RenderAndroidView.gestures}
-///
-/// See also:
-///
-///  * [UiKitView] which is a widget that is used to show a UIView.
-///  * [PlatformViewsService] which is a service for controlling platform views.
-class RenderUiKitView extends RenderBox {
-  /// Creates a render object for an iOS UIView.
+/// Superclass for common fucntionality of platform view RenderBoxes
```

Comment:
```suggestion
/// Superclass for common functionality of platform view RenderBoxes
```

Reply:
nit: doc sentences should be grammatically correct and end with a `.`, https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-correct-grammar